### ISSUE TITLE
Issue with gammapy jupyter commands with jupiter_core v4.11

### ISSUE THREAD

**Issue by registerrier** [gammapy/gammapy#4030]
The notebook CI build is broken recently. This is likely caused by a recent version of jupyter_core package (v4.11).

A fresh install of the environment-dev will break `gammapy jupyter` e.g.:
```
gammapy jupyter --src docs/tutorials/api/makers.ipynb test
INFO:gammapy.scripts.jupyter:   ... EXECUTING: makers.ipynb in docs/tutorials/api
/Users/terrier/Code/anaconda3/envs/gammapy-dev/bin/python3.8: No module named jupyter
ERROR:gammapy.scripts.jupyter:Error executing notebook: makers.ipynb in docs/tutorials/api
INFO:gammapy.scripts.jupyter:   ... FAILED makers.ipynb
INFO:gammapy.scripts.jupyter:
```

And in python:
```
In [1]: import jupyter_core

In [2]: jupyter_core.__version__
Out[2]: '4.11.0'

In [3]: import jupyter
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 import jupyter

ModuleNotFoundError: No module named 'jupyter'
```

Reverting to jupyter_core v4.10 solves the issue.

**Comment by registerrier** [comment]
This is a known issue https://github.com/jupyter/jupyter_core/issues/275 and has been solved in v4.11.1. Our CI is now working fine.
Closing now.

**Comment by adonath** [comment]
Thanks for investigating!
